elastic/kibana
# [Upgrade Assistant] Disable UI by default in 8.0

Fixes https://github.com/elastic/kibana/issues/115684

### Summary

This PR disables Upgrade Assistant UI by default, since for 8.0 it isn't functional yet. I also skipped the associated tests with it since they don't need to run anymore.

**How to test**
1. Start up elasticsearch, `yarn es snapshot --license=trial`
3. Start up Kibana, `yarn start` and navigate to `Stack Management`
4. Verify that `Upgrade Assistant` is not visible
5. Verify that setting `xpack.upgrade_assistant.ui.enabled: true` in `config/kibana.yml` enables `Upgrade Assistant` and the coming soon prompt is shown.
